ivy/func_wrapper.py

```diff
@@ -847,6 +848,10 @@ def _handle_out_argument(*args, out=None, **kwargs):
         """
         if out is None or is_compos_fn:
             return fn(*args, out=out, **kwargs)
+
+        if _is_variable(out):
```

Comment:
probably we can do a `handle_out_in_backend = handle_out_in_backend and _is_variable(out)` instead as that's a bit cleaner